### PR TITLE
[PROGRAMMES-6932] Gallery page throws an internal error AGAIN

### PR DESCRIPTION
**Pull request by tomwinnington**
Ticket:
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6932

If a gallery has no image pid then it throws an error

Link:
http://sandbox.bbc.co.uk/programmes/p00v7b7j

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Controller/FindByPid/GalleryController.php [hunk 1]
```diff
@@ -23,7 +23,10 @@ public function __invoke(
         $this->setContextAndPreloadBranding($gallery);
         $siblingLimit = 4;
         $images = $imagesService->findByGroup($gallery);
-        $image = $this->getFirstImage($imagePid, $images);
+        $image = null;
+        if ($imagePid) {
```
Comment (BjornTwachtmann) [inline src/Controller/FindByPid/GalleryController.php]: This change is unncessary, it's already checked in the function, it's just the check in the template you added that's needed (I think, please check I'm not talking shite though)

File: config/services.yaml [hunk 2]
```diff
@@ -28,7 +28,7 @@ parameters:
     app.idtquiz.service.class: App\ExternalApi\IdtQuiz\Service\IdtQuizService
     app.favourites.baseurl: 'https://www.bbc.co.uk/modules/plugin/musicfavourite/init'
     app.favourites.service.class: App\ExternalApi\FavouritesButton\Service\FavouritesButtonService
-    app.isite.baseurl: 'https://api.live.bbc.co.uk/isite2-content-reader'
+    app.isite.baseurl: 'https://api.test.bbc.co.uk/isite2-content-reader'
```
Comment (benolot) [inline config/services.yaml]: Make sure not to commit this

File: src/Controller/Schedules/ByDayController.php [hunk 3]
```diff
@@ -255,6 +254,7 @@ private function getOtherNetworks(Service $service, NetworksService $networksSer
             'bbc_two',
             'bbc_three',
             'bbc_four',
+            'bbc_scotland',
```
Comment (benolot) [inline src/Controller/Schedules/ByDayController.php]: I think this was from another ticket

**Pull request merged**